avocado-setup.py

```diff
@@ -189,6 +189,26 @@ def get_avocado_bin():
         return avocado_binary
 
 
+def pip_install():
+    """
+    install package using pip
+    """
+    logger.info("install packages via pip interface")
+
+    pip_cmd = 'pip%s' sys.version_info[0]
```

Comment:
`     pip_cmd = 'pip%s' % sys.version_info[0] `

Reply:
ah miss that as while making changes i miss that :(